wormhole-connect/src/config/mainnet/wrappedTokens.ts

```diff
@@ -0,0 +1,621 @@
+export const MAINNET_WRAPPED_TOKENS = {
```

Comment:
I wonder if we should re-enable the checkForeignAssetsConfig script until we can remove the wrapped tokens <network> files

Reply:
I guess we should